README.md

```
@@ -1,152 +1,158 @@
-
-For detailed architecture guidelines, see `AGENTS.md`.
\ No newline at end of file
+![image](https://github.com/user-attachments/assets/ecc94303-df23-46ae-87dc-66b040396e0b)
+
+If you have a suggestion that would make this better, please fork the repo and
+create a pull request. You can also
+[open issues](https://github.com/superset-sh/superset/issues).
+
+See the [CONTRIBUTING.md](CONTRIBUTING.md) for instructions and code of conduct.
+
+#### Contributors
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)